jvm/src/main/scala/com/nawforce/apexlink/org/CompletionProvider.scala

```diff
@@ -356,13 +364,42 @@ trait CompletionProvider {
       case None => emptyCompletions
     }
   }
+
+  private def getEmptyCreatorCompletionItems(
+    td: ApexFullDeclaration,
+    line: String
+  ): Array[CompletionItemLink] = {
+    // Strip 'id = n' from '... typeName id = new '
+    val trimmed = idAssignPattern.replaceFirstIn(line, "")
+    if (trimmed == line)
+      return emptyCompletions
+
```

Comment:
Kind of, VSCode asks for completion on the 'n' and continues to show the completion if you type 'e' etc. If you type something other than 'e' it will ask for completion again if we did not provide any matches the first time around.

We have a bit of a mismatch here in that when you press 'n' we give a reply as though the 'n' wasn't there because of how we interface to C3. We could post-process that to just returning things starting with 'n' but VSCode does that for us anyway. I don't think this causes any issues but not completely sure.